docker/docker_image_src.go

```diff
@@ -129,9 +129,42 @@ func (s *dockerImageSource) ensureManifestIsLoaded() error {
 	return nil
 }
 
+func (s *dockerImageSource) getExternalBlob(urls []string) (io.ReadCloser, int64, error) {
+	var (
+		resp *http.Response
+		err  error
+	)
+	for _, url := range urls {
+		resp, err = s.c.makeRequestToResolvedURL("GET", url, nil, nil, -1)
```

Comment:
Right, we just need the client w/o auth here because we may need auth certificates against the give  foreign server. We may probably need a new method in the client to make calls w/o authentication but still honor cert-path and the like for the given server we're connecting to here.